modules/private-clusters-setting-dns-private.adoc

```diff
@@ -4,13 +4,22 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="private-clusters-setting-dns-private_{context}"]
-= Setting DNS to private
+= Configuring DNS records to be published in a private zone
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+For all {product-title} clusters, whether public or private, DNS records are published in a public zone by default.
+
+You might use a public zone for DNS records in cases where you want to have a private cluster while still allowing clients to look up names for applications running on that cluster. For example, if an organization has machines that connect to the public internet, and establish VPN connections for certain private IP ranges in order to connect to private services. The DNS lookups from these machines use the public DNS to determine the private addresses of those services, and then connect to the private addresses over the VPN.
+
+Conversely, you might use a private zone for DNS records to avoid exposing sensitive information contained in DNS records to the public. DNS records can include information such as internal domain names, internal IP addresses, or the number of clusters at an organization. You might also simply have no need to publish records publicly and want to avoid unnecessarily publishing DNS records.
+
+If all the clients that need to connect to services within the cluster use a private DNS service that has the DNS records from the private zone, then there is no need to have a public DNS record for the cluster.
+
+After you deploy a cluster, you can modify its DNS to use only a private zone by modifying the `DNS` custom resource (CR).
+Modifying the `DNS` CR in this way means that any DNS records that are created are not published to public DNS servers, which keeps knowledge of the DNS records isolated for internal users. This should be done when you configure the cluster to be private, or if you never want DNS records to be publicly resolvable.
```

Comment:
I suggest adding "subsequently" to emphasize that the Operator doesn't automatically delete DNS records from the zone when you remove the zone from the CR, and changing "should" to "can" as it isn't strictly necessary to do for private clusters:
```suggestion
Modifying the `DNS` CR in this way means that any DNS records that are subsequently created are not published to public DNS servers, which keeps knowledge of the DNS records isolated to internal users. This can be done when you configure the cluster to be private, or if you never want DNS records to be publicly resolvable.
```